charts/common/templates/_helpers.tpl

```diff
@@ -4,8 +4,8 @@
 
 {{- define "labels" }}
 app: {{ empty .Values.releaseName | ternary .Release.Name .Values.releaseName }}
-shortname: {{ .Values.shortname }}
-team: {{ .Values.team }}
+app_id: {{ .Values.shortname }}
+team: team-{{ .Values.team }}
```

Comment:
They might already use this aswell ?

Reply:
If we want to change it, maybe make it a breaking version and give a heads-up in slack

Reply:
Maybe we should keep `team` as is, and `shortname` and rename the new with some prefix like `entur_team` and `entur_app_id`. 
I think the most relevant case for these is in the cost reports.